Poseiden-skeleton/src/main/java/com/nnk/springboot/services/RatingService.java

```diff
@@ -31,7 +31,7 @@ public void update(Integer id, Rating ratingToUpdate) {
         ratingRepository.save(ratingToUpdate);
     }
 
-    public void delete(Integer id) {
+    public void deleteById(Integer id) {
         ratingRepository.deleteById(id);
```

Comment:
Normalement il faut faire une recherche avant de supprimer.
```suggestion
        ratingRepository.findById(id).orElseThrow(() -> new NotFoundException("Rating with id " + id + " not found"))
        ratingRepository.deleteById(id);
```